beacon_node/network/src/sync/backfill_sync/mod.rs

```diff
@@ -898,6 +942,54 @@ impl<T: BeaconChainTypes> BackFillSync<T> {
         Ok(())
     }
 
+    /// Retries partial column requests within the batch by creating new requests for the failed columns.
+    #[instrument(parent = None,
+        fields(service = "backfill_sync"),
+        name = "backfill_sync",
+        skip_all
+    )]
```

Comment:
Ah I don't think this needs to be a root span. I've actually covered sending `DataColumnByRange requests` in the other tracing PR. I think the retry will just show up in the parent range request span, so we can delete this one.

https://github.com/sigp/lighthouse/pull/7831/files#diff-2a2c481817feb9f76f2ff94088fd03e7217ff4d108065649b179fb1aa1c38b2aR485-R493